File: api/treinaweb-flask-api-gastos-pessoais/api/entidades/conta.py
class Conta():
    def __init__(self, titulo, saldo):
        self.__titulo = titulo
        self.__saldo = saldo

    @property
    def titulo(self):
        return self.__titulo

    @titulo.setter
    def titulo(self, titulo):
        self.__titulo = titulo

    @property
    def saldo(self):
        return self.__saldo

    @saldo.setter
    def saldo(self, saldo):
        self.__saldo = saldo

File: app/treinaweb-flutter-gerenciador-gastos-pessoais/lib/models/conta.dart
import 'package:gerenciador_gastos_pessoais/models/transacao.dart';

class Conta {
  int id;
  String titulo;
  double saldo;
  List<Transacao> transacoes;

  Conta({this.id, this.titulo, this.saldo});

  Map<String, dynamic> toMap() {
    return {
      'id': id,
      'titulo': titulo,
      'saldo': saldo
    };
  }

  Map<String, dynamic> toJson() {
    return {
      'titulo': titulo,
      'saldo': saldo
    };
  }

  Conta.fromJson(Map json) {
    var list = json["transacoes"] as List;
    List<Transacao> transacaoList = list.map((e) => 
      Transacao.fromJson(e)).toList();
    id = json["id"];
    titulo = json["titulo"];
    saldo = json["saldo"];
    transacoes = transacaoList;
  }

  Conta.fromMap(Map map) {
    id = map["id"];
    titulo = map["titulo"];
    saldo = map["saldo"];
  }
}